syllabus/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils.timezone import datetime
+
+
+class Link(models.Model):
+    link = models.CharField(max_length=500, null=False, blank=False, default="https://incwellventure.com")
+    link_for = models.CharField(max_length=500, null=False, blank=False, default="Github")
+
+    def __str__(self):
+        return f'{self.link}: {self.link_for}'
+
+
+class Course(models.Model):
```

Comment:
@shakyasaijal  courses and mentor have many to many relationships have you maintained that?